app/controllers/posts_controller.rb

```diff
@@ -58,17 +60,22 @@ def destroy
 
   # GET /about
   def about
-    # @post = Post.find(???)
+    begin
+      @post = Post.unscoped.find_by! static_id: :about
+    rescue ActiveRecord::RecordNotFound
+      @post = Post.unscoped.create! static_id: :about, title: 'Rólunk oldal tartalma', short_description: '[Nem szükséges kitölteni]'
```

Comment:
I think we don't need to unscope here.

Reply:
Yes, I removed `.unscoped` from before the `.create!`, but the `.find_by!` needs the `.unscoped` as the simple `find_by!` would only look for Posts in the default_scope (where there's no `:about` Post). Tested it manually.